modules/sizing-requirements-for-gitops.adoc

```diff
@@ -28,3 +28,45 @@ Optionally, you can also use the ArgoCD custom resource with the `oc` command to
 ----
 oc edit argocd <name of argo cd> -n namespace
 ----
+
+[id="sizing-requirements-for-argocd-redis_{context}"]
+== Sizing requirements for argocd-redis
+
+During the capacity planning stage for your application in the {gitops-title} Operator, you must ensure that an adequate amount of resources, such as memory, CPU, and storage, are allocated for the `argocd-redis` pod.
+
+The default memory limit for the Redis pod might not be enough to manage a large number of resources. In these instances, you must increase the memory limit, monitor the memory metrics, and change the memory configuration while the application deployment scales up.
+
+The following command shows the example of the memory configuration for a Redis pod in the `openshift-gitops` namespace:
+[source,terminal]
+----
+oc get argocd -n openshift-gitops openshift-gitops -o json | jq '.spec.redis.resources'
+----
+
+.Example Output
+[source,terminal]
+----
+{
+    "limits": { #<1>
+        "cpu": "500m",
+        "memory": "256Mi"
+  },
+  "requests": { #<2>
+    "cpu": "250m",
+    "memory": "128Mi"
+  }
+}
+----
+<1> The highest resource limit threshold allocated to the pod.
+<2> The lowest resource limit threshold allocated to the pod.
+
+The following example command changes the memory configuration for a Redis pod. The highest resource limit threshold is set to 8 GiB and the lowest is set to 256 MiB.
+[source,terminal]
+----
+oc patch argocd -n openshift-gitops openshift-gitops --type json -p '[{"op": "replace", "path": "/spec/redis/resources/limits/memory", "value": "8Gi"}, {"op": "replace", "path": "/spec/redis/resources/requests/memory", "value": "256Mi"}]'
```

Comment:
```suggestion
$ oc patch argocd -n openshift-gitops openshift-gitops --type json -p '[{"op": "replace", "path":  \
  "/spec/redis/resources/limits/memory", "value": "8Gi"}, {"op": "replace", "path": \
  "/spec/redis/resources/requests/memory", "value": "256Mi"}]' \
```
See https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#backslash-for-long-lines-of-code

Reply:
Fixed.